### ISSUE TITLE
Findfeatures - PVL group [Results] Error

### ISSUE THREAD

**Issue by lwellerastro** [DOI-USGS/ISIS3#4375]
**ISIS version(s) affected**: 4.4.0

**Description**  
Findfeatures runs and generates output files but produces the following errors when doing so:
```
  Group = Error
    Program = findfeatures
    Code    = 1
    Message = "Unable to find PVL group [Results]"
    File    = PvlObject.cpp
    Line    = 137
  End_Group
```
This is the case for isis4.4.0 but not prior versions.  I compared the output files from isis4.3.0 and isis4.4.0 and everything is essentially the same (processing times for steps vary) including the output networks which are identical. 

Other then the error message I'm not sure if this negatively effects the output.
I'll run under isis4.3.0 in case I'm missing something.

**How to reproduce**  
See test data under /work/users/lweller/Isis3Tests/FindFeatures/PVLResults/
The images N1590648465_1.sharp.dck.cub and N1604395063_1.sharp.dck.cub are input to the command found in run_ff.cmd. 
Everything else in the directory is output from the program.

The subdirectory Isis4.3.0/ contains the input/output for a no error run under isis4.3.0.

**Comment by rvwagner** [comment]
This bug does not seem to have been fixed in ISIS 5.0.1.  `findfeatures` still returns a non-0 exit code and reports an error of `**ERROR** Unable to find PVL group [Results] in PvlObject.cpp at 122` even if it produces valid results.  The output seems fine, but the error code on exit makes it difficult to use findfeatures in a script.

Test case: (Note that I've tried this on actual image data and gotten the same result, pun mostly intended)
```
makecube lines=100 samples=100 bands=1 to=t.cub
fakecube from=t.cub to=t2.cub op=ch
findfeatures match=t2.cub from=t2.cub algorithm=sift/sift
```
```
Group = MatchSolution
  Matcher      = sift/sift/BFMatcher@NormType:NORM_L2@CrossCheck:false
  MatchedPairs = 1
  Efficiency   = 0.066090712742981
End_Group
**ERROR** Unable to find PVL group [Results] in PvlObject.cpp at 122.
```

I do seem to actually have ISIS 5.0.1, and the changelog claims this has been fixed:

```
$ cat $ISISROOT/version
5.0.1 # Public version number
2021-06-17 # Release date
stable # Release stage (alpha, beta, stable)

$ grep -i findfeatures $ISISROOT/CHANGELOG.md
- Fixed logging in FindFeatures where we were trying to get a non-existent Pvl group from the Pvl log. [#4375](https://github.com/USGS-Astrogeology/ISIS3/issues/4375)
 - findfeatures now properly resets the input images between algorithm runs when running in multi-algorithm mode.
 - findfeatures now reports an error instead of seg faulting when it tries to invert an empty matrix. [#557](https://github.com/USGS-Astrogeology/ISIS3/issues/557)
```

**Comment by lwellerastro** [comment]
Can confirm locally - the bug still exits under isis5.0.1

**Comment by jessemapel** [comment]
It looks like the fix for this (#4399) didn't get moved over to the 5.0.1 branch. We will need to pull a 5.0.2 that contains this fix and any other bug fixes we've merged since 5.0.2. I'm not sure of when we'll have time to get to that, but I'll make sure I update over here when we do.

**Comment by victoronline** [comment]
Any update on if this will be cherry picked back to 5.0.1 and if it will be included in 6.0?

**Comment by victoronline** [comment]
I see that it was included in 5.0.2. I assume this minor tick won't be available in conda until the next release. Is this correct?

**Comment by rvwagner** [comment]
It does seem that this has been fixed in ISIS 5.0.2 and ISIS 6.0.0 RC1.  My test case above runs correctly in both, with no error message and a successful return code.

**Comment by jessemapel** [comment]
This is available in both 5.0.2 and 6.0.0-RC1 (will be in 6.0.0 upon full release).
